api/variables.go

```diff
@@ -183,6 +183,44 @@ func (vars *Variables) GetVariableItems(path string, qo *QueryOptions) (Variable
 	return v.Items, qm, nil
 }
 
+// RenewLock renews the lease for the lock on the given variable. It has to be called
+// before the lock's TTL expires or the lock will be automatically released after the
+// delay period.
+func (vars *Variables) RenewLock(v *Variable, qo *WriteOptions) (*VariableMetadata, *WriteMeta, error) {
+	v.Path = cleanPathString(v.Path)
+	var out VariableMetadata
+
+	wm, err := vars.client.put("/v1/var/"+v.Path+"?renew-lock", v, &out, qo)
```

Comment:
oh!  this was a bug I found during hack week

```suggestion
	wm, err := vars.client.put("/v1/var/"+v.Path+"?lock-renew", v, &out, qo)
```